### DISCUSSION TITLE
Assistance with configuring Nextcloud Assistant with ChatGPT

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#5264] [Questions] [answered]
Greetings,
I am currently running Nextcloud Hub 8 (v9.5.1) and would like to integrate Nextcloud Assistant with my paid subscription to ChatGPT. I have encountered some issues during the setup, particularly with the following error message:

> *RuntimeException Error during request to ExApp (context_chat_backend): ExApp `context_chat_backend` not found. LanguageModel call using provider Nextcloud Assistant Context Chat Provider failed.*

I would greatly appreciate a detailed, step-by-step guide on how to properly configure Nextcloud Assistant to use ChatGPT as the AI provider. Specifically, I am looking for instructions on:

1. How to correctly configure Nextcloud Assistant to use ChatGPT.
2. The necessary API settings or permissions required to ensure smooth integration with ChatGPT.
3. Any additional modules or configurations that need to be enabled for Context Chat and other AI functionalities.
4. How to resolve the error message mentioned above.
5. Recommendations for checking log files to identify specific issues with AI integration.

To provide additional context, I am paying for a monthly subscription to ChatGPT and have generated an API key from OpenAI, but I am unsure if I have set everything up correctly within Nextcloud.

Thank you in advance for your assistance. I look forward to your guidance.

Best regards,  
Tony

**Comment by szaimen** [comment, marked as answer]
This will be answered at https://help.nextcloud.com/t/assistance-with-nextcloud-assistant-setup-and-openai-api-integration/203941
